conans/client/graph/graph.py

```diff
@@ -36,15 +36,14 @@ def __init__(self, ref, conanfile, recipe=None, path=None):
         self.recipe = recipe
         self.remote = None
         self.binary_remote = None
-        self.build_require = False
-        self.private = False
         self.revision_pinned = False  # The revision has been specified by the user
 
         # A subset of the graph that will conflict by package name
         self.public_deps = None  # {ref.name: Node}
         # all the public deps only in the closure of this node
         # The dependencies that will be part of deps_cpp_info, can't conflict
         self.public_closure = None  # {ref.name: Node}
+        self.transitive_closure = None  # {ref.name: Node}
```

Comment:
Can you provide a comment with the intention of this new member?

Reply:
Yes, I will provide a comment now, but after feedback I will rename these variables for something more meaningful.

This is the set of dependencies that need to be propagated to consumers downstream (not build requires, private, or transitive of build-requires or private).

Reply:
Added comment.